dm/loader/lightning.go

```diff
@@ -234,15 +235,15 @@ func (l *LightningLoader) restore(ctx context.Context) error {
 		}
 		cfg.Routes = l.cfg.RouteRules
 
-		// TODO lightning checkpoint will support s3 in other pr
-		if storage.IsS3Path(l.cfg.LoaderConfig.Dir) {
-			cfg.Checkpoint.Enable = false
-		} else {
-			cfg.Checkpoint.Driver = lcfg.CheckpointDriverFile
-			cpPath := filepath.Join(l.cfg.LoaderConfig.Dir, lightningCheckpointFileName)
-			cfg.Checkpoint.DSN = cpPath
-			cfg.Checkpoint.KeepAfterSuccess = lcfg.CheckpointOrigin
+		cfg.Checkpoint.Driver = lcfg.CheckpointDriverFile
```

Comment:
why hard code to file checkpoint ?  seems lightning support use `CheckpointDriverMySQL` checkpoint?

Reply:
To reduce the complexibility that checkpoint exists but dump files doesn't exist

Reply:
our configuration about `loaders` do not support other checkpoint too.